compose/cpp-and-twogtp/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Builds an image that can run KataGo's C++ binaries as well as run twogtp.
+#
+# Since this image performs a multi-stage build based on
+# humancompatibleai/goatttack:cpp, that image should be rebuilt before
+# rebuilding this image.
+#
+# Example usage of twogtp:
+#     mkdir --parents /outputs/sgfs && \
+#     /gogui/bin/gogui-twogtp \
+#       -black '/engines/KataGo-raw/cpp/katago gtp -config /go_attack/configs/gtp-raw.cfg -model <victim model>' \
+#       -white '/engines/KataGo-custom/cpp/katago gtp -config /go_attack/configs/gtp-amcts.cfg -model <adversary model> -victim-model <victim model>' \
+#       -alternate -auto -games <number of games> -komi 6.5 -maxmoves 1600 \
+#       -sgffile /outputs/sgfs/game -verbose \
+#     2>&1 | tee --append /outputs/twogtp.log
+FROM humancompatibleai/goattack:cpp as cpp
+
+COPY ./controllers/gogui /gogui
+WORKDIR /gogui
+# Modify ubuntu_setup.sh to play nice with Docker build
+RUN sed --in-place 's/sudo apt install/apt install --assume-yes/' ./ubuntu_setup.sh
+RUN apt-get update --quiet && ./ubuntu_setup.sh && apt-get clean && rm -rf /var/lib/apt/lists/*
+WORKDIR /
```

Comment:
I wanted to combine `compose/cpp/Dockerfile` and `compose/transfer/twogtp/Dockerfile`, but the obvious thing of doing `FROM humancompatibleai/goattack:cpp as cpp` and then copying the twogtp executable from `humancompatibleai/goattack:twogtp` didn't work because there were some missing installations (missing Java). I'm just copy-and-pasting the twogtp installation commands from `compose/transfer/twogtp/Dockerfile` since the list of commands is short